src/codegen/utilities/attribute_helpers.py

```diff
@@ -233,6 +233,10 @@
     "PhysicalChannel": "physical_chan",
 }
 
+ATTRIBUTES_RETURN_FILE_PATH_TYPE = ("logging_file_path",)
+
+ATTRIBUTES_WITH_FILE_PATH_PARAMETER = ("logging_file_path",)
```

Comment:
Is there a good reason for these to be separate? Since you are updating both the getter and setter for the one file path property, I think you should merge these.

Reply:
I thought there could be some attributes have either return or accept file path type. It's fixed.